bim_webviewer/bim_web_viewer.md

```diff
@@ -3447,6 +3447,34 @@ GUI
 
 ---
 
+### Set Hotkeys Enabled
+
+<p class="heading-link-container"><a class="heading-link" href="#set-hotkeys-enabled"></a></p>
+
+```js
+setHotkeysEnabled(enabled);
+```
+
+#### Description
+
+Allows you to enable or disable hotkeys. Hotkeys are keyboard shorcuts that will turn on/off tools. For example, pressing "X" will turn on the X-Ray mode when hotkeys are enabled. When disabled, the hotkeys will not work and pressing "X" will have no effect.
```

Comment:
Hotkeys are not necessarily just for turning on/off tools. For example, undo/redo, navigate to home view, and pressing Esc to exit out of things are included (currently anyways). 

I think this still makes sense if we just remove that part. I don't think "hotkeys" as a concept requires explanation. Thoughts?

```suggestion
Allows you to enable or disable hotkeys. For example, pressing "X" will turn on the X-Ray mode when hotkeys are enabled. When disabled, the hotkeys will not work and pressing "X" will have no effect.
```

Reply:
Agreed, this looks like a good suggestion. I was questioning it too. Its in the PR!